app/controllers/api/functions.php

```diff
@@ -1926,9 +1951,38 @@
         $execution->setAttribute('responseBody', $executionResponse['body'] ?? '');
         $execution->setAttribute('responseHeaders', $headers);
 
-        $response
-            ->setStatusCode(Response::STATUS_CODE_CREATED)
-            ->dynamic($execution, Response::MODEL_EXECUTION);
+        $acceptTypes = \explode(', ', $request->getHeader('accept', 'multipart/form-data'));
+        $isJson = false;
```

Comment:
As I said before, here we need opposite logic. Notice this is using JSON only if you provided JSON accept header. And anything else is multipart.

We need JSON by default, and only if you specified multipart, then we do multipart.

Reply:
Re-opening this based on recent discussion on Discord.

Reply:
It wasnt addressed yet.

Reply:
Done